assets/src/components/AssignmentTable.js

```diff
@@ -54,6 +54,13 @@ const styles = theme => ({
     paddingTop: '22px',
     paddingLeft: '5px',
     verticalAlign: 'middle'
+  },
+  randaom: {
```

Comment:
Is this a misspelling of random? What is this class for? I don't also see it used anymore after the most recent commit.

Reply:
It is not used. This is still WIP